### PR TITLE
fixes#6124 <ADMINS> argument removed

### LINKED ISSUE #6124: Hide `ockam space create` option for adding `ADMINS`
# Current behavior

When you run `ockam space create --help` the following output is displayed.

Image

The `<ADMINS>` argument can be passed in via the command line.

# Desired behavior

The `<ADMINS>` argument should be removed and not displayed in the help.

Please take a look at this [file](https://github.com/build-trust/ockam/blob/develop/implementations/rust/ockam/ockam_command/src/space/create.rs#L24) to get started.

---

<i>We love helping new contributors! ❤️
If you have questions or need help as you explore, please join us on [Discord](https://discord.gg/bewvnm6zqS). If you're looking for other issues to contribute to, please checkout our [good first issues](https://github.com/build-trust/ockam/labels/good%20first%20issue).</i>

### PR DESCRIPTION
**Pull request by AdityaRanjanJha**
fix(rust): <ADMINS> argument removed from space create command

## Current behavior

The <ADMINS> argument has been removed and now its not displayed in the help.

## Proposed changes

By removing the pub admins: Vec<String> from file location: implementations/rust/ockam/ockam_command/src/space/create.rs

### CODE REVIEW

**Comment by mrinalwadhwa** [comment]
@AdityaRanjanJha Thank you for spending time on this, before we can merge please

- [x] Accept the Ockam [Contributor License Agreement](https://github.com/build-trust/ockam-contributors/blob/main/CLA.md) by adding your Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam/blob/develop/.github/CONTRIBUTORS.csv) file in a separate new pull request to the [build-trust/ockam](https://github.com/build-trust/ockam) repository.

The easiest way to do this is to [edit the CONTRIBUTORS.csv](https://github.com/build-trust/ockam/edit/develop/.github/CONTRIBUTORS.csv) file in the github web UI and create a PR, this will mark the commit as verified.

**Comment by mrinalwadhwa** [comment]
Thank you @AdityaRanjanJha, CLA accepted in #6167 
We'll wait for @nazmulidris to get a chance to review before we merge this.

**Comment by adrianbenavides** [comment]
Hey @AdityaRanjanJha, here are some tips to pass the failing CI jobs:
- Our repo doesn't accept merge commits. That is, to update the base branch, use `git rebase develop`
- We enforce some conventions when writing git messages. In this case, "fixes#6124 <ADMINS> argument removed" could be rewritten as "fix(rust): `<ADMINS>` argument removed from `space create` command"
- Make sure to run both `cargo fmt` and `cargo clippy` as you introduce your changes. You will find that there are currently some compiling errors in your code

Let me know if you have any question with any of the above 👍

**Review by adrianbenavides — CHANGES_REQUESTED** [review]
Hey @AdityaRanjanJha, here are some tips to pass the failing CI jobs:

- Our repo doesn't accept merge commits. That is, to update the base branch, you have to use `git rebase develop` instead of `git merge develop`
- We enforce some conventions when writing git messages. In this case, `fixes#6124 argument removed` could be rewritten as `fix(rust): <ADMINS> argument removed from space create command`
- Make sure to run both `cargo fmt` and `cargo clippy` as you introduce your changes. You will find that there are currently some compiling errors in your code


Let me know if you have any question with any of the above 👍

**Comment by adrianbenavides** [comment]
Hey @AdityaRanjanJha I think the PR got closed because you force-merged latest develop on your branch. If you want to try again, make sure you add your changes in a separate branch (not in your fork's develop branch) and that you use `git rebase develop` instead of `git merge develop`.

**Pull request closed without merging**
